framework/package.json

```diff
@@ -20,8 +20,8 @@
 		"url": "https://github.com/LiskHQ/lisk-sdk/issues"
 	},
 	"engines": {
-		"node": ">=10.4 <=10",
-		"npm": ">=6"
+		"node": ">=12.13.0 <=12",
```

Comment:
let's make it consistent for all the packages
Also, packages in the elements need to be updated too

Reply:
Fixed